src/test/mir-opt/separate_const_switch.rs

```diff
@@ -0,0 +1,24 @@
+#![feature(control_flow_enum)]
+
+use std::ops::ControlFlow;
+
+// EMIT_MIR separate_const_switch.too_complex.SeparateConstSwitch.diff
```

Comment:
Can you also emit the constprop diff and the Precodegen MIR ? 
It will help picture what we actually feed to llvm.